lib/serve/index.ts

```diff
@@ -134,42 +133,10 @@ export class LisaServeApplicationStack extends Stack {
             description: 'URI for LISA Serve API',
         });
 
-        // Register all models
-        const registeredModels: RegisteredModel[] = [];
-
-        // Create ECS models
-        for (const modelConfig of config.ecsModels) {
-            if (modelConfig.deploy) {
-                // Create ECS Model Construct
-                const ecsModel = new EcsModel(this, createCdkId([getModelIdentifier(modelConfig), 'EcsModel']), {
-                    config: config,
-                    modelConfig: modelConfig,
-                    securityGroup: vpc.securityGroups.ecsModelAlbSg,
-                    vpc: vpc.vpc,
-                });
-
-                // Create metadata to register model in parameter store
-                const registeredModel: RegisteredModel = {
-                    provider: `${modelConfig.modelHosting}.${modelConfig.modelType}.${modelConfig.inferenceContainer}`,
-                    // modelId is used for LiteLLM config to differentiate the same model deployed with two different containers
-                    modelId: modelConfig.modelId ? modelConfig.modelId : modelConfig.modelName,
-                    modelName: modelConfig.modelName,
-                    modelType: modelConfig.modelType,
-                    endpointUrl: ecsModel.endpointUrl,
-                };
-
-                // For textgen models, add metadata whether streaming is supported
-                if (modelConfig.modelType === ModelType.TEXTGEN) {
-                    registeredModel.streaming = modelConfig.streaming!;
-                }
-                registeredModels.push(registeredModel);
-            }
-        }
-
         // Create Parameter Store entry with registeredModels
         this.modelsPs = new StringParameter(this, createCdkId(['RegisteredModels', 'StringParameter']), {
```

Comment:
We can remove this if it's not going to be used anymore, unless we need it in other areas.